pkgs/development/python-modules/fabric/default.nix

```diff
@@ -32,6 +33,7 @@ buildPythonPackage rec {
     paramiko
     cryptography
     decorator
+    pynacl
```

Comment:
`paramiko` is missing `pynacl`. This was fixed by https://github.com/NixOS/nixpkgs/commit/50d865aa192645e10064173422b394a12aecdfea during the last Python mass update (https://nixpk.gs/pr-tracker.html?pr=345326).

Reply:
@fabaff any idea why it is hanging and refused to get out of staging-next to main for so long?

Reply:
@takeda staging-next needs to be build by [hydra](https://hydra.nixos.org/jobset/nixpkgs/staging-next) first (and problems need to be worked out), before it can be merged. Since these are mass rebuilds, they can take a while.